### PR TITLE
Add Node.js v8 to Travis CI

### PR DESCRIPTION
**Pull request by ryysud**
Add support for Node.js v8.
Updated .travis.yml.

### CODE REVIEW

**Comment by abouthiroppy** [comment]
I think we can drop 0.12 and 0.10.

**Comment by ryysud** [comment]
@abouthiroppy I think so too.

**Comment by ryysud** [comment]
Please follow up on this 🙇

**Comment by abouthiroppy** [comment]
@substack @maxogden What do you think?

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 language: node_js
 node_js:
+  - "8"
   - "7"
```
Comment (stevemao) [inline .travis.yml]: could you remove `7` and add `9`?

**Comment by stevemao** [comment]
Note this will be a breaking change

**Comment by ryysud** [comment]
@stevemao I fix it.

**Comment by goto-bus-stop** [comment]
@ryysud sorry for the wait, curious if you could check out the test failure? it looks like it's a bug in the traceur version used by the `yield.js` test, maybe bumping the `es6ify` dependency in our package.json could work?

**Comment by goto-bus-stop** [comment]
i'll remove es6ify after merging. thanks for the patch!

**Comment by ljharb** [comment]
fwiw I think dropping node < 4 was a very very bad move, and I think that should be reverted.

**Comment by ryysud** [comment]
🙇

**Comment by ryysud** [comment]
@ljharb Sorry. Thank you for your Pull Request 🙇

**Pull request merged**